docs/running/pkcs11/_index.md

```diff
@@ -216,34 +216,36 @@ KEY_3_TYPE='rsa-2048'
 # export PKCS11_SPY_PATH='/usr/lib64/pkcs11/pkcs11-spy.so'
 # export PKCS11_SPY_PATH='/usr/lib/arm-linux-gnueabihf/pkcs11/pkcs11-spy.so'
 
+PKCS11_TEST_PATH="$PWD/target/x86_64-unknown-linux-gnu/debug/pkcs11-test"
```

Comment:
Nit pick, document the paths for arm as well.

Reply:
Done.